### PR TITLE
git-annotate: add Korean translation

### PR DESCRIPTION
**Pull request by ellijs**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.ko/common/git-annotate.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# git annotate
+
+> 각각의 파일란에 커밋한 해쉬와 마지막 작성자를 보여 줍니다. 깃 어노테이트 보다 많이 사용되는 깃 블레임을 살펴 보세요. 깃 어노테이트는 다른 버전 관리 시스템에 친숙한 분들께 제공됩니다.
+> 더 많은 정보: <https://git-scm.com/docs/git-annotate>.
+
+- 각각의 라인에 작성자의 이름과 커밋 해쉬를 앞쪽에 더하여 파일 출력:
+
+`git annotate {{path/to/file}}`
```
Comment (navarroaxel) [inline pages.ko/common/git-annotate.md:8]: Please translate the placeholders, like `path/to/file`, for get this easier for non-English speakers.

Comment (kbdharun) [reply pages.ko/common/git-annotate.md:8]: @ellijs Refer #8820 if necessary

File: pages.ko/common/git-annotate.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# git annotate
+
+> 각각의 파일란에 커밋한 해쉬와 마지막 작성자를 보여 줍니다. 깃 어노테이트 보다 많이 사용되는 깃 블레임을 살펴 보세요. 깃 어노테이트는 다른 버전 관리 시스템에 친숙한 분들께 제공됩니다.
+> 더 많은 정보: <https://git-scm.com/docs/git-annotate>.
+
+- 각각의 라인에 작성자의 이름과 커밋 해쉬를 앞쪽에 더하여 파일 출력:
+
+`git annotate {{경로/목적지/파일이름}}`
```
Comment (navarroaxel) [inline pages.ko/common/git-annotate.md:8]: It's different from `경로/대상/파일` at https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/common-arguments.md. Could we standardize this?

Comment (ellijs) [reply pages.ko/common/git-annotate.md]: Addressed 👍 Thank you @navarroaxel

**Review by navarroaxel — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]

File: pages.ko/common/git-annotate.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# git annotate
+
+> 각각의 파일란에 커밋한 해쉬와 마지막 작성자를 보여 줍니다. 깃 어노테이트 보다 많이 사용되는 깃 블레임을 살펴 보세요. 깃 어노테이트는 다른 버전 관리 시스템에 친숙한 분들께 제공됩니다.
```
[the remark this answers is not in the document]
Comment (kbdharun) [reply pages.ko/common/git-annotate.md:3]: Create a new PR to fix this #9061

**Pull request merged**